addon/components/power-select-multiple-with-create.js

```diff
@@ -5,15 +5,21 @@ export default PowerSelectWithCreate.extend({
   layout,
   powerSelectComponentName: 'power-select-multiple',
 
-  selectOrCreate(selection, select) {
-    let suggestion = selection.filter((option) => {
-      return option.__isSuggestion__;
-    })[0];
+  // Probably needs to be fixed upstream
+  triggerComponent: 'power-select-multiple/trigger',
+  beforeOptionsComponent: null,
```

Comment:
Without those two lines here, the PowerSelect is rendered with the default components for a single select

Reply:
This should be fixed by https://github.com/cibernox/ember-power-select/pull/1271 upstream.